pyfar/dsp/dsp.py

```diff
@@ -223,3 +224,75 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def deconv(measurement, excitation, **kwargs):
+    """Function to calculate the deconvolution of two signals.
+
+    Parameters
+    ----------
+    measurement : pyfar.Signal
+        The measurement signal, recorded after passing the device under test.
+    excitation : pyfar.Signal
```

Comment:
You should add that measurement is zero padded if it is shorter than excitation. (And add the opposite below to excitation)

Reply:
check